Makefile

```diff
@@ -0,0 +1,38 @@
+include ./Makefile.os
+include ./Makefile.docker
+include ./Makefile.maven
+
+PROJECT_NAME ?= kafka-bridge
+GITHUB_VERSION ?= master
+RELEASE_VERSION ?= latest
+
+ifneq ($(RELEASE_VERSION),latest)
+  GITHUB_VERSION = $(RELEASE_VERSION)
+endif
+
+all: java_package docker_build docker_push
+clean: java_clean
+release: release_prepare release_version release_helm_version release_maven $(SUBDIRS) release_docu release_single_file release_pkg release_helm_repo docu_clean
```

Comment:
we don't have helm. Not sure that all these above are existing targets following here.

Reply:
Yeah, that were leftovers - I cleaned them up. But I guess the release will anyway need some finishing up to package the docs etc. But that should be done later.